async-raft/tests/fixtures/mod.rs

```diff
@@ -115,6 +115,45 @@ impl RaftRouter {
         metrics
     }
 
+
+    pub async fn get_sto(&self, node_id: &NodeId) -> Arc<MemStore> {
+        let sto = self.routing_table.read().await.get(node_id).unwrap().clone().1;
+        sto
+    }
```

Comment:
```suggestion
    /// Get a handle to the storage backend for the target node.
    pub async fn get_storage_handle(&self, node_id: &NodeId) -> Result<Arc<MemStore>> {
        let sto = self.routing_table.read().await.get(node_id).context("could not find node in routing table")?.clone().1;
        Ok(sto)
    }

```

I'm making these suggestions because even in test code, it makes sense not to use unwrap. This helps to make it clear why a test might fail.